tests/test_schema_retrieval.py

```diff
@@ -26,29 +26,34 @@
 class TestSchemaRetrieval:
     "Test schema retrieval."
 
+    # We allow this to fail because the default remote might be down but it's not this library's issue.
+    @pytest.mark.xfail
```

Comment:
```suggestion
    @pytest.mark.xfail(reason="default remote might be down but it's not this library's issue", raises=ConnectionError)
```

Do you think we can use the `raises` parameter here? https://docs.pytest.org/en/latest/skipping.html#raises-parameter

Reply:
Good point, added